WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderSiteHeaderView.java

```diff
@@ -131,13 +130,15 @@ private void showBlogInfo(ReaderBlog blogInfo) {
         }
 
         try {
-            txtFollowCount.setText(
-                    String.format(getContext().getString(R.string.reader_label_follow_count), blogInfo.numSubscribers));
+            txtFollowCount.setText(String.format(
+                            LocaleManager.getSafeLocale(getContext()),
+                            getContext().getString(R.string.reader_label_follow_count),
+                            blogInfo.numSubscribers));
         } catch (ArithmeticException exception) {
-            // See https://github.com/wordpress-mobile/WordPress-Android/issues/5568
-            // In case of a "ArithmeticException: divide by zero", force the use of "US" locale to format the string.
-            txtFollowCount.setText(String.format(Locale.US, getContext().getString(R.string.reader_label_follow_count),
-                                                 blogInfo.numSubscribers));
+            txtFollowCount.setText(String.format(
```

Comment:
It looks like our try and catch are the same now. Maybe we don't need the try and catch anymore, but if we do, shouldn't they be different statements?